cluster-autoscaler/FAQ.md

```diff
@@ -94,6 +104,39 @@ They are also not particularly suited to use with Kubernetes in general.
 
 ****************
 
+# How to?
```

Comment:
I feel those questions don't fit into any existing section, but I'm not too happy with the name of this one. I'm happy to rename it if you have any better ideas. 

@mwielgus @aleksandra-malinowska